cli/src/command/account.rs

```diff
@@ -283,7 +283,50 @@ pub async fn burn_nft_command(account_handle: &AccountHandle, nft_id: String) ->
 
 // `balance` command
 pub async fn balance_command(account_handle: &AccountHandle) -> Result<(), Error> {
-    println_log_info!("{:#?}", account_handle.balance().await?);
+    let balance = account_handle.balance().await?;
+    println_log_info!("{balance:#?}");
+    println_log_info!("Claimable outputs: {}", balance.potentially_locked_outputs().len());
+
+    for output_id in
+        balance.potentially_locked_outputs().iter().filter_map(
+            |(output_id, unlockable)| {
+                if *unlockable { Some(output_id) } else { None }
+            },
+        )
+    {
+        if let Some(output_data) = account_handle.get_output(output_id).await {
+            let output = output_data.output;
+            let kind = match output {
+                Output::Nft(_) => "Nft",
+                Output::Basic(_) => "Basic",
+                Output::Alias(_) => "Alias",
+                Output::Foundry(_) => "Foundry",
+                Output::Treasury(_) => "Treasury",
```

Comment:
These output types will never be in the potentially_locked_outputs since they can't have SDR, expiration or timelock unlock conditions

Reply:
yeah, I figured that...we should just replace it with an 'unreachable!' then.